Set up Issue template

mihrab34 opened the pull request:
to close #694

mihrab34 commented:
@adamghill I have some commits from my `origin main branch`. My local branch is up to date with the upstream and origin. Is there a way I could clean the commit history up? I'm thinking `git rebase`

adamghill commented:
Yes! I think a rebase and force push should do it. If it doesn't work we can take a look at it tomorrow. 👍

mihrab34 commented:
@adamghill

adamghill commented on .github/workflows/coverage.yml:
Is this from the other PR? Maybe a rebase will clear this out?

mihrab34 replied:
yes, got that after merging my `origin main` with my current feature branch. Had to clean it up with rebase

adamghill reviewed (changes requested):
The issue templates look great! If you can clear out the extraneous commits that would be awesome, otherwise I might be able to clean them up. Let me know!

mihrab34 commented:
> The issue templates look great! If you can clear out the extraneous commits that would be awesome, otherwise I might be able to clean them up. Let me know!

@adamghill  I squashed all of them into 1 and fixed the part with the merge conflict.

adamghill reviewed (approved):
🚀
